### PR TITLE
fix(l10n): Use an inline `t` function to ensure l10n works as expected.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
`t` in `const {t} = BaseView` was being translated by Webpack/babel into
_base.t, which is not recognized by our l10n extraction script.

This changes all `t` declarations to use `const t = msg => msg` which
in local testing, has always kept the same name.

I tried using a module with a simple `module.export = msg => msg` and
import/requiring this using `import t from 'lib/t'`, but Webpack/babel
sometimes changed the `t` to `t_` which also is not recognized by
our extraction script.

issue #6725

### CODE REVIEW

**Comment by vladikoff** [comment]
Good find!

File: app/scripts/views/permissions.js [hunk 1]
```diff
@@ -183,14 +184,17 @@ var View = FormView.extend({
     // convert the permission names to HTML
     return sortedPermissionNames.map((permissionName) => {
       var permission = this._getPermissionConfig(permissionName);
+
+      permission.label = this.translate(permission.label);
```
[the remark this answers is not in the document]
Comment (shane-tomlinson) [reply app/scripts/views/permissions.js]: I think all permissions, if requested, are required, aren't they?

**Comment by shane-tomlinson** [comment]
@vladikoff - mind an r on this? It would be nice to get into train-126 if possible.

**Comment by SorinaFlorean** [comment]
Same as https://github.com/mozilla/fxa-content-server/issues/6728#issuecomment-445223963.

**Comment by shane-tomlinson** [comment]
For whatever reason, the strings were not correctly extracted for train-126. I'm going to have to ask for a re-check with train-127. Sorry @SorinaFlorean!

**Comment by SorinaFlorean** [comment]
Works as expected on production.

**Pull request merged**